src/System.Net.Http/src/System/Net/Http/Managed/ManagedHandler.cs

```diff
@@ -239,6 +239,36 @@ public SslProtocols SslProtocols
             }
         }
 
+        public TimeSpan ConnectionTimeout
```

Comment:
The name does need work. It sounds like the timeout for establishing connections.

Reply:
Perhaps `ConnectionLifetime` or `ConnectionLifetimeDuration` ?

Reply:
> The name does need work.

Yes.  As I noted in the PR description, it's a placeholder.  Suggestions welcome.  PooledConnectionTimeout?  ConnectionPoolingTimeout?  ConnectionLifetimeTimeout? Etc.

Reply:
Did we finalize this in the API review?